project-base/storefront/components/Blocks/Product/ProductsList/ProductsListContent.tsx

```diff
@@ -22,6 +22,7 @@ type ProductsListProps = {
     swipeHandlers?: SwipeableHandlers;
     className?: string;
     classNameProduct?: string;
+    isMiniSlider?: boolean;
```

Comment:
According to previous comment `isWithSimpleCards`...?

Reply:
No problem, renamed.